pkg/open/open.go

```diff
@@ -29,3 +29,22 @@ func Browser(url string) error {
 
 	return nil
 }
+
+// LacksBrowser determines if no browser is set in linux
+func LacksBrowser() bool {
+	if runtime.GOOS != "linux" {
```

Comment:
also checking only `linux` here may be a bit too restrictive as this would technically work on other platforms that have `xdg-utils` installed as well, could just flip this to only attempt to try if it's not `darwin` or `windows`